incubator/vault/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for vault.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: vault
+  tag: 0.8.1
+  pullPolicy: IfNotPresent
+service:
+  name: vault
+  type: ClusterIP
+  port: 8200
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  hosts:
+    - chart-example.local
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
+# A YAML representation of a final vault config.json file.
+# See https://www.vaultproject.io/docs/configuration/ for more information.
+vaultmode:
+# Only used to enable dev mode. When in dev mode, the rest of this configuration
+# is ignored. See https://www.vaultproject.io/intro/getting-started/dev-server.html
+# for more information.
+  dev: true
```

Comment:
Could you document in the README that this setup is insecure by default, and what is the configuration needed to setup TLS on an internal cluster that cannot use Lets Encrypt?

Reply:
Running Vault in dev mode is insecure by design.

Reply:
Understood, but given the fact that vault is supposed to store secrets, I think it's worth the effort to keep people from running an insecure vault.

Now that I think about it, we should also add a warning in the status after we install vault if we are in dev mode.